starfish/experiment/experiment.py

```diff
@@ -89,11 +107,69 @@ def name(self) -> str:
     def image_types(self) -> Set[str]:
         return set(self._images.keys())
 
-    def get_image(self, item: str, crop_parameters: Optional[CropParameters]=None) -> ImageStack:
-        return ImageStack.from_tileset(self._images[item], crop_parameters)
+    def show_aligned_image_groups(self):
+        """
+        Describe the aligned subgroups for each Tileset in this FOV
+
+        ex.
+            {'primary': 'Group 0:  <starfish.ImageStack (8, 2, 1, 205, 405))>',
+             'nuclei': 'Group 0:  <starfish.ImageStack (1, 1, 1, 205, 405))>'}
+
+        Means there are two tilesets in this FOV, (primary and nuclei) each tileset only
+        has one aligned subgroup.
+
+        ex.
+            {'primary': 'Group 0:  <starfish.ImageStack (4, 2, 1, 205, 405))>
+                         Group 1:  <starfish.ImageStack (4, 2, 1, 205, 405))>'}
+
+        Means there is one tileset in this FOV, (primary), it has two different aligned
+        coordinate groups with 4 rounds in each
 
-    def __getitem__(self, item: str) -> ImageStack:
-        return self.get_image(item)
+        """
+        all_groups = dict()
+        for name, groups in self.aligned_coordinate_groups.items():
+            y_size = self._images[name].default_tile_shape[0]
+            x_size = self._images[name].default_tile_shape[1]
+            info = '\n    '.join(
+                f" Group {k}: "
+                f" <starfish.ImageStack "
+                f"""({len(v._permitted_rounds) if v._permitted_rounds else 1, 
+                    len(v._permitted_chs) if v._permitted_chs else 1,
+                 len(v._permitted_zplanes) if v._permitted_zplanes else 1, y_size, x_size})>"""
+                for k, v in enumerate(groups)
+            )
+            all_groups[name] = f'{info}'
+        return all_groups
+
+    def iterate_image_type(self, image_type: str):
+        for aligned_group, _ in enumerate(self.aligned_coordinate_groups[image_type]):
+            yield self.get_image(item=image_type, aligned_group=aligned_group)
+
+    def get_image(self, item, aligned_group: int = 0,
+                  x_slice: Optional[Union[int, slice]] = None,
+                  y_slice: Optional[Union[int, slice]] = None,
+                  ) -> ImageStack:
+        """
```

Comment:
I'm confused about this. I thought that crop_params were previously used to crop in pixel space, but my understanding here is that the `aligned_coordinate_groups` are in coordinate space. 

In any case, would like very robust description here with examples -- this function will be called by users.

Reply:
aligned_coordinate_groups are just crop params that describe what (permitted) r/ch/z are in each aligned group. If a user wants to grab an aligend group and also crop on x/y pixel space, they'd pass in these slices.

Reply:
There's probably a compelling reason to remove the whole crop-on-load when we move to dask.

Reply:
Is this really `get_aligned_group` now, and not `get_image`?